docs/options.md

```diff
@@ -539,8 +573,7 @@ CIBW_DEPENDENCY_VERSIONS: ./constraints.txt
 
 Shell command to run tests after the build. The wheel will be installed
 automatically and available for import from the tests. To ensure the wheel is
-imported by your tests (instead of your source copy), tests are run from a
-different directory. Use the placeholders `{project}` and `{package}` when
+imported by your tests (instead of your source copy), **tests are not run from your project directory**. Use the placeholders `{project}` and `{package}` when
```

Comment:
```suggestion
imported by your tests (instead of your source copy), **tests are not run from
your project directory**. Use the placeholders `{project}` and `{package}` when
```

Reply:
Does this keep the **markdown** markup? It's not highlighted anymore by GitHub, now?

Reply:
That's just the "suggestion" view, which is some sort of diff without parent language (guessing it guesses Python here?)